appeal/job_handler.go

```diff
@@ -57,3 +65,40 @@ func (h *JobHandler) RevokeExpiredAccess() error {
 	log.Println(string(result))
 	return nil
 }
+
+func (h *JobHandler) NotifyAboutToExpireAccess() error {
+	daysBeforeExpired := []int{7, 3, 1}
```

Comment:
will make this configurable in the upcoming PR together with making the notification messages configurable